src/content-cli-push.ts

```diff
@@ -42,6 +42,11 @@ class Push {
                 "Specify this option if you want to push all Data models into one already existing pool with provided ID",
                 null
             )
+            .option(
```

Comment:
What happens if we don't have a spaceKey here?

Reply:
It will select the default space, and if the default space is also not there it id going to throw the exception.